measures/SimulationOutputReport/measure.rb

```diff
@@ -236,8 +236,10 @@ def run(runner, user_arguments)
 
     # ELECTRICITY
 
-    report_sim_output(runner, "total_site_electricity_kwh", electricity.total_end_uses[0], "GJ", elec_site_units)
-    report_sim_output(runner, "net_site_electricity_kwh", electricity.total_end_uses[0] - electricity.photovoltaics[0], "GJ", elec_site_units)
+    unless include_enduse_subcategories
+      report_sim_output(runner, "total_site_electricity_kwh", electricity.total_end_uses[0], "GJ", elec_site_units)
+      report_sim_output(runner, "net_site_electricity_kwh", electricity.total_end_uses[0] + electricity.photovoltaics[0], "GJ", elec_site_units)
```

Comment:
We probably don't need both net_site_electricity_kwh and total_site_electricity_kwh.
And, we probably want total_site_electricity_kwh, regardless of include_enduse_subcategories is true or false.

Reply:
We talked. We decided to:
* Remove all the net site columns
* Leave total columns for both true and false
* Total columns become net of pv

The only difference between false and true is that interior equipment is replaced with each of its components.